src/main/java/com/meetup/server/user/domain/User.java

```diff
@@ -59,4 +69,25 @@ public void updateAgreement(boolean personalInfoAgreement, boolean marketingAgre
     public void updateNickname(String nickname) {
         this.nickname = nickname;
     }
+
+    public void updateToWithdraw() {
```

Comment:
P3) User의 탈퇴 기능이므로 update를 붙이지 않아도 괜찮을 거 같습니다!

```suggestion
    public void withdraw() {
```